src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/SVConcordance.java

```diff
@@ -94,29 +124,42 @@
 @DocumentedFeature
 public final class SVConcordance extends AbstractConcordanceWalker {
 
-    public static final String UNSORTED_OUTPUT_LONG_NAME = "do-not-sort";
-
     @Argument(
             doc = "Output VCF",
             fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME
     )
-    private GATKPath outputFile;
+    protected GATKPath outputFile;
 
+    /**
+     * Expected format is tab-delimited and contains columns NAME, RECIPROCAL_OVERLAP, SIZE_SIMILARITY, BREAKEND_WINDOW,
+     * SAMPLE_OVERLAP. First line must be a header with column names. Comment lines starting with
+     * {@link TableUtils#COMMENT_PREFIX} are ignored.
+     */
     @Argument(
-            doc = "Skip output sorting. This can substantially reduce memory usage.",
-            fullName = UNSORTED_OUTPUT_LONG_NAME
+            doc = "Configuration file (.tsv) containing the clustering parameters for each group",
+            fullName = GroupedSVCluster.CLUSTERING_CONFIG_FILE_LONG_NAME,
+            optional = true
     )
-    private boolean doNotSort;
+    public GATKPath strataClusteringConfigFile;
+
+    @Argument(fullName = SVClusterWalker.MAX_RECORDS_IN_RAM_LONG_NAME,
+            doc = "When writing VCF files that need to be sorted, this will specify the number of records stored in " +
+                    "RAM before spilling to disk. Increasing this number reduces the number of file handles needed to sort a " +
+                    "VCF file, and increases the amount of RAM needed.",
+            optional=true)
+    public int maxRecordsInRam = 10000;
```

Comment:
You mentioned decreasing this had a big impact on performance. Should the default be lowered?

Reply:
In local tests yes. I was going to wait until we had more data on that but I think it is safer to lower to 100.